sdk/iot/modelsrepository/src/DTDL.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft.
+// Licensed under the MIT license.
+
+interface Contents {
+  "@type"?: string;
```

Comment:
yes it is. The DTDL type used here is kept internal away from the API surface though because it's a psuedo-proxy for the DTDL parser that will be released later.

Reply:
type will be made required tho.